src/pages/timecard/Timecard.jsx

```diff
@@ -154,22 +154,31 @@ const updateTimeEntryContextData = async (
   validateServiceErrors(setServiceError, async () => {
     const timeEntriesResponse = await getTimeEntries(timeEntriesParams);
 
+    const timeCardStart = dayjs(date).startOf('day').add(1, 'minute');
+    const timeCardEnd = dayjs(date).endOf('day');
+
     if (timeEntriesResponse.data.items?.length > 0) {
-      const existingTimeEntries = timeEntriesResponse.data.items.map(
+      const filteredTimeEntries = timeEntriesResponse.data.items.filter(
         (timeEntry) => {
-          return new ContextTimeEntry(
-            timeEntry.id,
-            timeEntry.actualStartTime,
-            timeEntry.actualEndTime ? timeEntry.actualEndTime : '',
-            timeEntry.timePeriodTypeId,
-            timeEntry.finishNextDay ??
-              isFinishTimeOnNextDay(
-                formatTime(timeEntry.actualStartTime),
-                formatTime(timeEntry.actualEndTime)
-              )
+          return !(
+            dayjs(timeEntry.actualEndTime).isBefore(timeCardStart) ||
+            dayjs(timeEntry.actualStartTime).isAfter(timeCardEnd)
```

Comment:
I know we spoke about this during the mob programming session, but now that I can see the finished filter
I **think** we may be able to include it all in the jpaRest request filter. Lets catchup and take another look to see if possible.
(Above comment becomes redundant if this can be done)

Reply:
resolving as the additional filtering has now been removed.